android/linting/build.gradle.kts

```diff
@@ -0,0 +1,15 @@
+plugins {
+    id("java-library")
+    id("org.jetbrains.kotlin.jvm")
+    //id("ktlint-kotlin-common")
```

Comment:
Can this be removed since it's commented out?

Reply:
Yes